specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -3774,6 +3774,10 @@
           "type": "object",
           "description": "The JSON Path of the Nested Array that is going to do cross-apply. Type: object (or Expression with resultType object)."
         },
+        "mapComplexValuesToString": {
+          "type": "object",
+          "description": "Whether to map complex (arrray and object) values to simple strings in json format. Type: boolean (or Expression with resultType boolean)."
```

Comment:
Why type boolean in description rather just make this field type of boolean? 
Is there any particular reason for `"type": "object"`?

Reply:
Customer use dynamic value through ADF pipeline parameterization, in this case, the value in the payload is an object which represent the parameterized expression, and the result of the expression has a boolean value.